Extensions/DialogueTree/dialoguetools.js

```diff
@@ -725,3 +725,16 @@ gdjs.dialogueTree.loadState = function(inputVariable) {
     console.error('Failed to load state from variable:', inputVariable, e);
   }
 };
+
+/**
+ * Clear the current State of the Dialogue Parser.
+ */
+gdjs.dialogueTree.clearState = function() {
+  try {
+    gdjs.dialogueTree.runner.visited = {};
+    gdjs.dialogueTree.runner.variables.data = {};
+    });
```

Comment:
typos - can you double check the extension is running?

Reply:
I am having trouble starting it now after upgrading gd to the latest master
![Screenshot_2020-05-13_19-10-55](https://user-images.githubusercontent.com/6495061/81842973-d447a000-9544-11ea-9508-dd3b7fb54c59.png)

I think there is an issue raised for it, but I have no idea how to fix it

Reply:
I added it to the example project, so as to be able to test it, but play testing with the extension now is broken for some reason

Reply:
nevermind, I forgot a } when applying review notes. Let me better test it :) added it to the example